core/common/src/main/java/alluxio/util/io/PathUtils.java

```diff
@@ -224,6 +224,16 @@ public static String temporaryFileName(long nonce, String path) {
   }
 
   /**
+   * Determines whether the given path is a temporary file name generated by Alluxio.
+   *
+   * @param path the path to check
+   * @return whether the given path is a temporary file name generated by Alluxio
+   */
+  public static boolean isTemporaryFileName(String path) {
+    return path.matches("^.*\\.alluxio\\.0x[0-9A-F]{16}\\.tmp$");
```

Comment:
Will this regex work if we go through the url encoder with something that needs to be encoded? (ie. is the encoding a 1 char - 1 char replacement)